api/infrastructure/handler/cart/getcart.go

```diff
@@ -0,0 +1,40 @@
+package cart
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/danielgtaylor/huma/v2"
+	"shanepee.com/api/domain"
+	"shanepee.com/api/infrastructure/handler"
+)
+
+type GetCartOutput struct {
+	Body []*domain.ArtToy
+}
+
+func (h *CartHandler) RegisterGetCart(api huma.API) {
+	huma.Register(api, huma.Operation{
+		OperationID: "get-cart",
+		Method:      http.MethodGet,
+		Path:        "/v1/cart",
+		Tags:        []string{"Cart"},
+		Summary:     "Get Cart",
+		Description: "Retrieve the user's cart",
+		Security: []map[string][]string{
+			{"sessionId": {}},
+		},
+	}, func(ctx context.Context, i *struct{}) (*GetCartOutput, error) {
+		userID := handler.GetUserID(ctx)
+		if userID == nil {
+			return nil, handler.ErrAuthenticationRequired
+		}
+		cart, err := h.cartSvc.GetCartByOwnerID(ctx, *userID)
+		if err != nil {
+			return nil, handler.ErrIntervalServerError
+		}
```

Comment:
```suggestion
		cart, err := h.cartSvc.GetCartByOwnerID(ctx, *userID)
		if err != nil {
			return nil, handler.ErrIntervalServerError
		}
```

handle not found duey